### ISSUE TITLE
Accessibility errors

### ISSUE THREAD

**Issue by closeyetfar** [internetarchive/bookreader#72]
Do you plan on making the Bookreader pass the WAVE accessibility tool? There is missing alt text, empty buttons and 1 contrast error.

<img width="1176" alt="screen shot 2018-05-26 at 8 01 20 am" src="https://user-images.githubusercontent.com/1972016/40575871-102dc786-60bb-11e8-9780-e8027a7ae91f.png">

**Comment by skorasaurus** [comment]
Since this issue was created a few years ago, several important accessibility issues remain: 

- The clickable controls (located at the bottom) for a book should be able to also be done using only a keyboard. Currently, they are very limited. Only Zoom in/out and changing pages can be done through the keyboard. 

For example, if a user does not have a mouse/trackpad or cannot use one, they cannot toggle from one to two page viewing, or read the book aloud using the audioplayer built into the bookreader ! 

- The buttons should also be [focuseable elements](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard#clickable_elements_must_be_focusable_and_should_have_interactive_semantics). This would provide a visual indication that the current active setting/element is and this also provide an audible notification which element is currently active and can be toggled by the user. 

 I tested using https://archive.org/details/birdbookillustra00reedrich (Which according to the readme as of this writing, should be using 5.0). 

Tested using Chrome 90 and Firefox 89.
